test_journey.py

```
@@ -0,0 +1,80 @@
+    pdf_content = create_mock_pdf()
+    files = {"file": ("demo_test.pdf", BytesIO(pdf_content), "application/pdf")}
+    data = {
+        "voice_provider": "openai",
+        "voice_type": "alloy",
+        "reading_speed": 1.0,
+        "include_summary": False,
+        "conversion_mode": "full"
+    }
+
+    try:
+        response = requests.post(f"{API_BASE_URL}/jobs", headers=headers, files=files, data=data)
```

Ruff (0.14.11)

Probable use of `requests` call without timeout

(S113)